src/manifest.json

```diff
@@ -6,7 +6,9 @@
   "icons": {
     "16": "icons/icon_generic_16.png",
     "48": "icons/icon_chrome_48.png",
-    "128": "icons/icon_chrome_128.png"
+    "128": "icons/icon_chrome_128.png",
+    "256": "icons/icon_chrome_256.png",
```

Comment:
Could you point me to the documentation where it points for sizes larger then 128?

Reply:
https://developer.apple.com/documentation/safariservices/safari_web_extensions/creating_a_safari_web_extension It's in preparation.